app/views/part/list.scala.html

```diff
@@ -12,11 +12,63 @@
     }
     -->
     </style>
+    @helper.javascriptRouter("jsRoutes")(routes.javascript.PartsController.list)
     @views.html.datatables()
+    <script type="text/javascript">
+    $(document).ready(function() {
+        var tableObject = $('table.dataTable');
+        var table = tableObject.DataTable();
+        var exportSelectedLink = $("#export-selected");
+        var selectAll = $("#select-all");
+
+        var updateDownloadLink = function(){
+            var selectedRows = table.rows(".selected", { filter: 'applied' }).data();
+            var ids = selectedRows.map(function(row){
+                return $(row[0]).text(); // extract id from id markup
+            });
+            if (ids.length > 0) {
+                var downloadUrl = jsRoutes.controllers.PartsController.list(ids).url;
+                exportSelectedLink.attr("href", downloadUrl);
+                toggleDownloadLink(true);
+            } else {
+                toggleDownloadLink(false);
+            }
+        };
+
+        var toggleDownloadLink = function(enabled) {
+          if(enabled) {
+            exportSelectedLink.removeClass("disabled");
+            exportSelectedLink.removeAttr("disabled");
+          } else {
+            exportSelectedLink.attr("disabled", "disabled");
+          }
+        }
+
+	    exportSelectedLink.attr("disabled", "disabled");
```

Comment:
This initialization should probably be higher up with the var declarations for visibility.